pages.ja/common/7zr.md

```diff
@@ -1,17 +1,17 @@
 # 7zr
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> .7zファイルをアーカイブタイプをサポートする
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
+> `7z`と似ていますが、`7zr`は`.7z`形式のみをサポートしています。
```

Comment:
```suggestion
> `7z` と似ていますが、`7zr` は `.7z` 形式のみをサポートしています。
```

Reply:
Thank you for the review! I've applied the suggestion.